flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+      type: object
+      properties:
+        refresh_period_r:
+          x-field-pattern:
+            description: >-
+              The refresh timeout period R used to generate this message;in milliseconds.
+            format: integer
+            default: 0
+            length: 32
+            features: [count]
+          x-field-uid: 1
+    Flow.RSVP.PathObjectsClassExplicitRoute:
+      description: >-
+          C-Type is specific to a class num.
+      type: object 
+      properties:
+        c_type: 
+          $ref: '#/components/schemas/Flow.RSVP.PathObjectsClassExplicitRouteCType'
+          x-field-uid: 1 
+    Flow.RSVP.PathObjectsClassExplicitRouteCType:
+      description: >-
+        Object for EXPLICIT_ROUTE class and c-type is Type 1 Explicit Route (1).
+      type: object
+      properties:
+        choice:
+          type: string
+          default: type_1_explicit_route
+          x-field-uid: 1
+          x-enum:
+            type_1_explicit_route:
+              x-field-uid: 1
+        type_1_explicit_route:
+          $ref: '#/components/schemas/Flow.RSVP.PathExplicitRouteType1'
+          x-field-uid: 2
+    Flow.RSVP.PathExplicitRouteType1:
+      description: >-
+        Type1 Explicit Route has subobjects.
+        Currently supported subobjects are IPv4 prefix and 4-byte AS number.
+      type: object
+      properties:
+        subobjects:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/Flow.RSVP.Type1ExplicitRouteSubobjects'
+          x-field-uid: 1
+    Flow.RSVP.Type1ExplicitRouteSubobjects:
+      description: >-
+          Type is specific to a subobject.
+      type: object 
+      properties:
+        type: 
+          $ref: '#/components/schemas/Flow.RSVP.Type1ExplicitRouteSubobjectsType'
+          x-field-uid: 1 
+    Flow.RSVP.Type1ExplicitRouteSubobjectsType:
+      description: >- 
+        Currently supported subobjects are IPv4 address(1) and 4-byte AS number(5).
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: ipv4_prefix
+          x-enum:
+            ipv4_prefix:
+              x-field-uid: 1
+            four_byte_as_number:
+              x-field-uid: 2
+        ipv4_prefix:
+          $ref: '#/components/schemas/Flow.RSVP.PathExplicitRouteType1Ipv4Prefix'
+          x-field-uid: 2
+        four_byte_as_number:
+          $ref: '#/components/schemas/Flow.RSVP.PathExplicitRouteType1FourByteASNumber'
+          x-field-uid: 3
+    Flow.RSVP.PathExplicitRouteType1Ipv4Prefix:
+      description: >-
+        Class = EXPLICIT_ROUTE, Type1 ROUTE_RECORD C-Type = 1
+        Subobject: IPv4 Prefix, C-Type: 1
+      type: object
+      properties:
+        l_bit:
+          x-field-pattern:
+            description: >-
+              The L bit is an attribute of the subobject.The L bit is set if the subobject represents a loose hop in the explicit route.
```

Comment:
space before starting of new sentence, i.e space before The, in the starting 2nd sentence

Reply:
Done.